superset/db_engine_specs/mysql.py

```diff
@@ -50,6 +50,9 @@
 TEST_CONNECTION_HOST_DOWN_REGEX = re.compile(
     "Can't connect to MySQL server on '(?P<hostname>.*?)'."
 )
+TEST_CONNECTION_UNKNOWN_DATABASE_REGEX = re.compile(
+    "Unknown database '(?P<database>.*?)'."
+)
```

Comment:
are these going to be the same errors that we'll hit when "creating" a db? Should we be prefixing them all with "TEST?" or it that too specific?

Reply:
would it be better as just connection_  ?

Reply:
I'll fix it in the Presto PR